packages/lab/stories/pill-next/pill-next.doc.stories.mdx

```diff
@@ -0,0 +1,112 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { PillNext } from "@salt-ds/lab";
+import { UserBadgeIcon } from "@salt-ds/icons";
+import {
+  CharacteristicUsage,
+  CSSClassTable,
+} from "css-variable-docgen-components";
+
+<Meta
+  title="Documentation/Lab/PillNext"
+  component={PillNext}
+  parameters={{
+    viewMode: "docs",
+  }}
+/>
+
+# PillNext
+
+Pill components provide multiple ways to label, tag or categorize content to
+allow users to trigger actions, make selections or filter results.
+
+## Usage
+
+Pill is a versatile, interactive and stand-alone component. It is often used to:
+
+#### Use When
+
+- **Tag content**
+  Use a Pill to organise content that may be mapped to more than one category. It
+  provides users with a way of distinguishing among different types of content and
+  categories.
+- **Trigger a quick action**
+  You can configure a Pill to trigger an immediate action—such as changing the view
+  of content that’s currently on display or navigating to a new page.
+- **Enable an action that’s subject to change**
+  Actions that may change depending on the context, and are not fixed, are best
+  represented by Pills rather than buttons. In a messaging application, for example,
+  Pills can be used to display quick stock responses which change so they’re appropriate
+  responses to a received message.
+- **Use title or sentence case** when labelling a Pill, depending on the nature of
+  the label. This maintains a visual distinction between Pills and Buttons,
+  which always have uppercase labels.
+
+## Basic Pill
+
+The default Pill is often used for tagging purposes, or to categorize content. Pill can be used to trigger an action that is subject to change as a result of user Pills or other factors. This Pill, for example, may be used to send an auto-generated quick response to a message. If a Pill enables an action that is fixed, and not subject to change, use a [Button](/?path=/docs/documentation-core-button--page) instead.
+
+**Best practices**
+
+- Position any Pill that you’re using to tag content _below_ or on the _right_
+  of the related content, so that it’s not the primary focus.
+- Never apply RAG (status) colors to the Pill component to differentiate between
+  categories. Use [Icon Descriptor](/?path=/docs/documentation-icons-icon--page) instead.
+
+<Canvas>
+  <Story id="lab-pill-next--default" />
+</Canvas>
+
+## Icon Descriptors
+
+Icons can be displayed on the left of a Basic or Closable Pill label to visually
```

Comment:
Closable pill isn't part of this version probably best to remove for now

Reply:
Good catch